openpype/settings/defaults/project_settings/photoshop.json

```diff
@@ -18,7 +18,7 @@
             "active": true
         },
         "ValidateNaming": {
-            "invalid_chars": "[ \\\\/+\\*\\?\\(\\)\\[\\]\\{\\}:,]",
+            "invalid_chars": "[ \\\\/+\\*\\?\\(\\)\\[\\]\\{\\}:,;]",
```

Comment:
Should these be in settings? If it's related to subset names then we should not allow any characters that are not in `SUBSET_NAME_ALLOWED_SYMBOLS` (`openpype.pipeline.create`).

Reply:
SUBSET_NAME_ALLOWED_SYMBOLS is allowing only ascii characters, we should also support utf-8 characters (as they might appear in layer names).
But thats a question for different PR.

Reply:
Created issue https://github.com/pypeclub/OpenPype/issues/2993